### ISSUE TITLE
Rule: mandatory braces for control flow statements

### ISSUE THREAD

**Issue by schalkms** [arturbosch/detekt#365]
Some guidelines include mandatory braces for control flow statements.
I think this rule should be deactivated per default.

**Comment by vanniktech** [comment]
What should we do about single line if statements? Since there's no ternatery operator.

**Comment by schalkms** [comment]
Maybe we should exclude single line if statements, but detect something like this.
```
if(condition)
  return 1
else
  return 0
```

**Comment by vanniktech** [comment]
Yup I agree. Also in addition those should not be flagged:

```kotlin
val x = if (condition) 5 else 4
if (condition) return 5
```

**Comment by schalkms** [comment]
Hey @jkaan detekt contributer here
This would definitely be a good starting point and we are gladly accepting contributed rules. 😊
Please take a look at one of the rules in the style package. You have to setup the rule, call it in the `StyleGuideProvider` and override the visit methods for the desired elements.
If you need assistance, do not hesitate to ask here or on our slack channel.
